apps/base/rucio-daemons/cms-rucio-daemons.yaml

```diff
@@ -228,6 +229,8 @@ config:
     tolerance: 0.01
     allowed_accounts: transfer_ops,ewv
     total_source: static
+    target_rse: None
```

Comment:
Things which are "None" should just be left out. This is not python, so None is going to become a string "None"